.gitignore

```diff
@@ -4,3 +4,6 @@ Cargo.lock
 #IntelliJ project files
 .idea
 *.iml
+
+CMakeLists.txt
```

Comment:
why does this add cmake files to the ignore file?

Reply:
https://github.com/rust-bitcoin/rust-secp256k1/pull/115/files#r292880923